scripts/kb_scripts/gwf_to_scs.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import os
+import shutil
+import sys
+
+from termcolor import colored
+from tqdm import tqdm
+
+from gwf_parser import GWFParser
```

Comment:
Move GWF parser and SCsWriter to separate folder in kb_scripts as support scripts. In kb_scripts should be only main scripts for prepare KB